src/backends/wasm_js.rs

```diff
@@ -7,32 +7,48 @@ pub use crate::util::{inner_u32, inner_u64};
 #[cfg(not(all(target_arch = "wasm32", any(target_os = "unknown", target_os = "none"))))]
 compile_error!("`wasm_js` backend can be enabled only for OS-less WASM targets!");
 
-use js_sys::{global, Uint8Array};
-use wasm_bindgen::{prelude::wasm_bindgen, JsCast, JsValue};
+#[cfg(target_feature = "atomics")]
+use js_sys::Uint8Array;
```

Comment:
It's better to move this import to the atomic `inner` function.

Reply:
Its required in the bindings on the bottom of the file as well.
I could:
- Move it into the atomic `inner()` function and use the full path when importing.
- Move the atomic only import into the atomic `inner()` function.
- Leave it as is.

Let me know.

Reply:
Ah, true. Then we can leave it as-is.

Reply:
BTW you also could just use `js_sys::Uint8Array` in both places without importing it.

Reply:
Right, let me know if you prefer that route.

Reply:
I prefer the code to be in the style maintainers prefer :)
I went ahead and applied your suggestion.